### PR TITLE
Make minor fixes to the Flux Monitor

### PR DESCRIPTION
**Pull request by felder-cl**
Use decimal.Zero, and use a type defined struct for the Job Run payload.  This
will improve safety and clarity in the codebase.

### CODE REVIEW

File: core/services/fluxmonitor/flux_monitor.go [hunk 1]
```diff
@@ -686,12 +693,10 @@ func (p *PollingDeviationChecker) createJobRun(nextPrice decimal.Decimal, nextRo
...
 }
 
-var dec0 = decimal.NewFromInt(0)
-
 // OutsideDeviation checks whether the next price is outside the threshold.
-func OutsideDeviation(curPrice, nextPrice decimal.Decimal, threshold float64) bool {
-	if curPrice.Equal(dec0) {
-		logger.Infow("Current price is 0, deviation automatically met", "currentPrice", dec0)
+func OutsideDeviation(nextAnswer decimal.Decimal, threshold float64) bool {
+	if curAnswer.IsZero() {
```
Comment (samsondav) [inline core/services/fluxmonitor/flux_monitor.go]: What am I missing here... I can't see where this variable comes from?

Comment (felder-cl) [reply core/services/fluxmonitor/flux_monitor.go]: nothing missing, this was a bad update

Comment (felder-cl) [reply core/services/fluxmonitor/flux_monitor.go]: Local changes were not included in this branch and written over, getting to it soon.

File: core/services/fluxmonitor/flux_monitor.go [hunk 2]
```diff
@@ -686,12 +693,10 @@ func (p *PollingDeviationChecker) createJobRun(nextPrice decimal.Decimal, nextRo
...
 }
 
-var dec0 = decimal.NewFromInt(0)
-
 // OutsideDeviation checks whether the next price is outside the threshold.
 func OutsideDeviation(curPrice, nextPrice decimal.Decimal, threshold float64) bool {
-	if curPrice.Equal(dec0) {
-		logger.Infow("Current price is 0, deviation automatically met", "currentPrice", dec0)
+	if curPrice.IsZero() {
+		logger.Infow("Current price is 0, deviation automatically met", "answer", decimal.Zero)
```
[the remark this answers is not in the document]
Comment (felder-cl) [reply core/services/fluxmonitor/flux_monitor.go]: that was carry over from the Flux Monitor PR which was recently merged.  @spooktheducks work.  I'll rebase and rework this one.

**Pull request merged**